ModelicaReference/package.mo

```diff
@@ -7013,6 +7013,22 @@ package Icons "Library of icons"
 end Icons;
 
 
+package AutoCompletion "Auto completion information for Modelica IDEs"
+  package Annotations "Auto completion information on annotations"
+      record experiment "Define default experiment parameters"
+        Real StartTime(unit = "s") "Default start time of simulation";
```

Comment:
Instead of hardcoding the unit here it should be using the proper type Modelica.SIunits.Time instead of Real. Same for the rest.

Reply:
It's a little bit weird actually. Because ModelicaReference does not depend on the standard library. So I think it's correct to define it as Real with a unit set manually...

Reply:
Good point. Didn't think about that.